example/app/(examples)/OutOfBandAssets.tsx

```diff
@@ -1,63 +1,84 @@
-import { SafeAreaView, ScrollView, StyleSheet, Text } from 'react-native';
+import * as React from 'react';
+import { SafeAreaView, ScrollView, StyleSheet, Text, View } from 'react-native';
 import Rive, { Fit, RNRiveError } from 'rive-react-native';
+import { Picker } from '@react-native-picker/picker';
 
 export default function StateMachine() {
+  const [uri, setUri] = React.useState('https://picsum.photos/id/372/500/500');
```

Comment:
The upside to `useState` is that it's how RN devs typically store and update values. The downside is that firing `setUri` will trigger a re-render. I've still never had something break due to a re-render, but it's not ideal. 

It might not matter since this is just an example, but the way I typically avoid re-renders in React is by using `useRef` instead. 

```
const urlRef = React.useRef('https://picsum.photos/id/372/500/500')

urlRef.current = 'https://picsum.photos/id/373/500/500'
```